docs/sequence_diagrams/notification-actionFlow.plantuml

```diff
@@ -0,0 +1,108 @@
+@startuml
+' declate title
+title 10.5. Notification Action Flow
+autonumber
+
+' declare actors
+collections "Notification-Topic" as TOPIC_NOTIFY
+boundary "Central Event Processor" as CEP
+control "Action Agent" as AA
+database "CEP Store" as DB
+
+box "Central Services" #LightYellow
+participant TOPIC_NOTIFY
+end box
+box "Central Event Processing Service" #LightGreen
+participant CEP
+participant AA
+participant DB
+end box
+
+' start flow
+Group Action Flow
+    activate CEP
+    CEP -> AA: Recieves an action event
+    activate AA
+    AA -> AA
+    note right of AA #lightGrey
+        Evaluate action event
+    end note
+
+    alt **Action Object** exist
+        AA -> DB: Request **Event Object**  details
+        activate DB
+        deactivate DB
+        hnote over DB
+            event
+        end hnote
+        note right of AA #LightGrey
+            **Event Object** details;
+                Action details,
+                Hub notification details,
+                DFSP notification details.
+        end note
+        AA -> AA
+        note right of AA #LightGrey
+            Prepare notification details
+        end note
+        AA -> DB: Request **Action Object** by same **Event Object** details
+        activate DB
+        deactivate DB
+        hnote over DB
+            action
+        end hnote
+
+        alt No previous **Action Object** with same event detail
+            AA -> DB: Store **Action Object**
+            activate DB
+            hnote over DB
+                action
+            end hnote
+            AA <-- DB: Return result **actionId**
+            deactivate DB
+            AA -> AA
+            note right of AA #LightGrey
+                Schedule reset repetition for
+                this **actionId** as per rules
+            end note
+            ref over AA :  Notification Scheduler Flow {[[https://github.com/mojaloop/central-event-processor/tree/master/docs/sequence_diagrams/notification-schedulerFlow.svg Scheduler Flow]]} \n
+
+            TOPIC_NOTIFY <- AA: Update topic with notification
+            note left of AA
+                {
+                    from field = "SYSTEM"
+                }
+            end note
+        else Previous **Action Object** with same event detail
+
+            alt Repetition and notification interval allowed
+                AA -> AA
+                note right of AA #LightGrey
+                    Increment repetition counter.
+                end note
+            AA -> DB: Update **Action Object**
+            else Repetition and notification interval not allowed
+            end
+            activate DB
+            deactivate DB
+'            AA -> AA
+'            note right of AA #LightGrey
+'                Schedule reset repetitions for **actionId**
+'                as per configuration rules.
+'            end note
+            TOPIC_NOTIFY <- AA: Update topic with notification
+            note left of AA
+                {
```

Comment:
same as previous

Reply:
Was applied as part of the previous request.
            note left of AA #yellow